src/main/java/com/app/backend/controller/DecedentController.java

```diff
@@ -0,0 +1,52 @@
+package com.app.backend.controller;
+
+import com.app.backend.model.Decedent;
+import com.app.backend.repository.DecedentRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDate;
+
+@RestController
+@RequestMapping("/api/decedent")
+@CrossOrigin("*")
+public class DecedentController {
+
+    @Autowired
+    private DecedentRepository decedentRepository;
+
+    @PostMapping("/add")
+    public ResponseEntity<String> addDecedent(@RequestBody Decedent decedent) {
+        ResponseEntity<String> validationResponse = validateDecedent(decedent);
```

Comment:
We should check if we at least started the endpoint. You can simply add System.out.println "Adding started for: " ...)